### ISSUE TITLE
TypeError("__init__() missing 1 required positional argument: 'params'")

### ISSUE THREAD

**Issue by DachenLiu** [ngruver/NOS#13]
{'ckpt_path': 'autodl-tmp/NOS/log_gaussian_train/autodl-tmp/NOS/log_gaussian_train/gaussian_5_sheet_discr_joint/models/best_by_valid/epoch=19-step=14220.ckpt',
 'data_dir': '/root/autodl-tmp/NOS/data',
 'exp_dir': '/root/autodl-tmp/NOS/sample_train/test_sample',
 'exp_name': 'test_sample',
 'guidance_kwargs': None,
 'infill_seeds_fn': '/root/autodl-tmp/NOS/infill_test_seeds.txt',
 'log_dir': '/root/autodl-tmp/NOS/sample_train',
 'max_epochs': 100,
 'max_seq_len': 300,
 'min_seq_len': 128,
 'model': {'_target_': 'seq_models.model.gaussian_diffusion.GaussianDiffusion', 'network': {'_target_': 'seq_models.model.gaussian_diffusion.GaussianDiffusionTransformer', 'in_channels': 16, 'vocab_size': '${vocab_size}', 'dropout': 0.0, 'bert_config_name': 'prajjwal1/bert-small', 'discr_stop_grad': True, 'target_channels': 1}, 'noise_schedule': {'_target_': 'seq_models.schedule.noise_schedule.GaussianDiffusionSchedule', 'timesteps': 1000, 'noise_schedule': 'cosine', 'noise_scale': 10}, 'optimizer': {'_target_': 'torch.optim.AdamW', 'lr': 0.0005}, 'lr_scheduler': {'_target_': 'transformers.get_linear_schedule_with_warmup', 'num_warmup_steps': 10, 'num_training_steps': '${max_epochs}'}},
 'ngpu': 1,
 'num_samples': 256,
 'project_name': 'guided_protein_seq',
 'seed': 0,
 'target_cols': ['ss_perc_sheet'],
 'trial_id': 0,
 'trim_strategy': 'randomcrop',
 'val_fn': 'val_iid.csv',
 'vocab_file': '/root/autodl-tmp/NOS/vocab.txt',
 'vocab_size': 30}
Error executing job with overrides: []
Error in call to target 'torch.optim.adamw.AdamW':
TypeError("__init__() missing 1 required positional argument: 'params'")
full_key: model.optimizer
How can I solve it ？
